src/config/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sitekey": "super secret key",
+  "GENERAL_API_URL": "http://localhost:8080",
+  "LOGGING_API_URL": "http://localhost:8081",
+  "MAILER_API_URL": "http://localhost:8082",
+  "RPC_API_URL": "http://localhost:8083"
+}
```

Comment:
I noticed that most of our `config.json` files are in gitignore and we have a `config.example.json` instead. Is there any reason why this isn't the case here?

Reply:
Ah yeah that'd be a conflict. I think I left this file in this commit to see if Travis would build, but now Travis runs `setup.py` which makes a `config.json` from the `config.example.json`. Will remove this file and force push